scripts/teamshow

```diff
@@ -24,12 +24,12 @@ import swsssdk
 import subprocess
 import sys
 from tabulate import tabulate
-from minigraph import parse_xml
 from natsort import natsorted
 from sonic_platform import get_machine_info
 from sonic_platform import get_platform_info
 
 PORT_CHANNEL_TABLE_PREFIX = "LAG_TABLE:"
```

Comment:
Now that you've added `PORT_CHANNEL_CFG_TABLE_PREFIX` below, I believe this constant should be renamed to `PORT_CHANNEL_APPL_TABLE_PREFIX` for consistency and to eliminate confusion.

Reply:
thanks, changes are done, please review